pkg/nuctl/command/import.go

```diff
@@ -256,6 +258,7 @@ Use --help for more information`)
 	}
 
 	cmd.Flags().StringSliceVar(&commandeer.skipProjectNames, "skip", []string{}, "Names of projects to skip (don't import), as a comma-separated list")
+	cmd.Flags().StringVar(&commandeer.skipProjectSelectors, "skip-selectors", "", "Selectors (label query) to filter projects on")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&commandeer.skipLabelSelectors, "skip-label-selectors", "", "Selectors (label query) to filter projects on")
```
If you don't mind.. because I don't think it's obvious nuctl inherits kubectl terminology self-evidently
also `skipLabelSelectors` because that same arg can be used to skip any other resource as well, you made it very generic 👍